pkg/vmprovider/providers/vsphere2/virtualmachine/configspec_test.go

```diff
@@ -6,11 +6,13 @@ package virtualmachine_test
 import (
 	goctx "context"
 
+	"github.com/google/uuid"
```

Comment:
Nit: sort this down below with the rest of the non dot imports (we should prb see if there's a linter for this).

Reply:
Done